app/views/goodsProfile/GoodsRecordsSearchResultView.scala.html

```diff
@@ -37,7 +37,7 @@
 <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">@messages("goodsRecordSearchResult.h1", totalRecords)</h1>
 
-    <p class="govuk-body">@messages("goodsRecordSearchResult.p1", searchString)</p>
+    <p class="govuk-body">@messages("goodsRecordSearchResult.p1", searchString.getOrElse(""))</p>
```

Comment:
I think this should change to something like the below, as the P tag will be on the page even when searchTerm is empty

```
searchString.map(searchString => <p class="govuk-body">@messages("goodsRecordSearchResult.p1", searchString))</p>

```